kbase-extension/static/kbase/js/widgets/function_output/kbaseAnnotatedMetagenomeAssembly.js

```diff
@@ -192,7 +192,7 @@ define ([
             var $input = $('<input type="text" class="form-control" placeholder="Search Features">');
             $input.prop('disabled', true);
 
-            var isLastQuery = function(result) {
+            var isLastQuery = function () {
```

Comment:
Can last query ever not be true? This doesn't seem to need to be a function right now.